### ISSUE TITLE
LT and RT half on and sticks aligned diagonally

### ISSUE THREAD

**Issue by Dayveid** [360Controller/360Controller#906]
### Type of Controller
PDP 048-082 for Xbox One

### OS Version
10.14.5

### Driver Version
0.16.6 through to 1.0.0 alpha 3

### Connection Method
wired

### Device Name and Info
>Device Name:
<br/>Product ID: 048-082
<br/>Vendor ID: PDP

### Response of Controller
I have been using various driver versions for a number of years now, with the same controller. Vibration has always worked, but lights never have (never been a problem for me)

### Response in Preference Pane
Randomly, half way through a game, the left stick aligned itself bottom left and the right stick to the top right. Both LT and RT remain half on. I have tried all the drivers listed above, and they all result in the same issue. I don't know if the controller itself has simply broken (but I doubt it), but, as I say, the same set up has worked for months and months seamlessly. The only difference is that, a few days ago, my Mac updated to the latest OS - but even then, everything was fine for a good few days.

### Detailed Explaination of Issue
It is probably worth noting that I have tried:
Enabling and re-enabling driver on each driver edition
Normalizing both sticks
'Pretend' to be Xbox 360 controller
Tinkering with the dead zones

Any ideas will be very welcome (even if it's simply buy a new controller!)

Thank you very much for your time.

David

**Comment by FranticRain** [comment]
I'm trying to think of why this might be happening. I looked at the flow of controller data and I can't come up with a reason this would happen in the driver. The values required to make this happen would be really strange, and I can't see it happening without some bad input from the device. Simply put, it would be nigh on impossible for the driver to do this without bad data.

It's hard to say for sure what the problem is. It's possible that the controller is returning some special packet that conflicts with the typical data input, causing this strange behavior. Like `0x02000200FFFF7FFF` is a really strange bit of data input. It would be great if we could make sure that the controller works perfectly with other device. Otherwise, I'm tempted to say that the controller is somehow messed up. An operating system update shouldn't interfere with the input, as the driver works in the kernel, before the OS can touch it.

Long story short, I have no idea what is happening here.
